sonar-plugin-api/src/main/java/org/sonar/api/ce/measure/MeasureComputer.java

```diff
@@ -142,6 +144,11 @@
        */
       void addMeasure(String metric, String value);
 
+      /**
+       * Return list of issues of current component.
+       */
+      List<? extends Issue> getIssues();
```

Comment:
I found no other to be able to directly return to list of DefaultIssue. Removing the wildcard needs to either return a list of wrapper of the DefaultIssue or to cast every DefaultIssue in Issue -> Both will require a lot of CPUs to process as the number of issues can be huge.